src/platform/packages/shared/kbn-es/src/serverless_resources/project_roles/security/roles.yml

```diff
@@ -846,6 +846,7 @@ endpoint_policy_manager:
         - feature_siemV3.endpoint_list_all
         - feature_siemV3.global_artifact_management_all
         - feature_siemV3.trusted_applications_all
+        - feature_siemV3.trusted_devices_all
```

Comment:
is this intentional here? if yes, do we want to add the new privilege only to `endpoint_policy_manager`?